security/pkg/platform/aws_test.go

```diff
@@ -165,7 +165,7 @@ func TestAwsGetServiceIdentity(t *testing.T) {
 			client: ec2metadata.New(unit.Session, &aws.Config{Endpoint: aws.String(server.URL + "/latest")}),
 		}
 
-		serviceIdentity, err := awsc.GetServiceIdentity()
+		serviceIdentity, _ := awsc.GetServiceIdentity()
```

Comment:
Doesn't this frustrate the linter since returned error is unchecked?

Reply:
Can you check this error?

Reply:
Thanks for pointing that out. Added a check for this error.